### DISCUSSION TITLE
Unable to use, request a refund

### DISCUSSION THREAD

**Discussion by jackbest96** [aclap-dev/video-downloadhelper#1078] [License & Purchase requests]
### What's your issue?

I can't download YouTube videos, it's stopping me from working

### The 10 first digits (only 10) of your license number or email address

21b04c549d

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
